runhouse/resources/hardware/cluster.py

```diff
@@ -1259,7 +1259,27 @@ def remove_conda_env(
 
     def download_cert(self):
         """Download certificate from the cluster (Note: user must have access to the cluster)"""
+        self.check_server()
```

Comment:
@dongreenberg is it fine for me to add this check here? `self.client` could be None otherwise